src/components/Garden/Feed/WrapToken.js

```diff
@@ -11,22 +10,31 @@ import {
   useTheme,
 } from '@1hive/1hive-ui'
 
+import Carousel from '@/components/Carousel/Carousel'
```

Comment:
Use the `Carousel` component to improve the wrap/unwrap/claim actions.

Reply:
```suggestion
import Carousel from '@components/Carousel/Carousel'
```